crates/turborepo/Cargo.toml

```
@@ -9,7 +9,7 @@ license = "MPL-2.0"
 # This is for the convenience of running daily dev workflows, i.e running
 # `cargo xxx` without explicitly specifying features, not that we want to
 # promote this as default backend.
-default = ["rustls-tls"]
+default = ["rustls-tls", "go-daemon", "run-stub"]
```

I'm guessing this is here by accident. If not, then it should be in it's own PR along with some testing/explanation.
```suggestion
default = ["rustls-tls"]
```